docset/winserver2022-ps/defender/Add-MpPreference.md

```diff
@@ -249,15 +274,16 @@ Accept wildcard characters: False
 ```
 
 ### -ThreatIDDefaultAction_Actions
-Specifies an array of the actions to take for the IDs specified by using the **ThreatIDDefaultAction_Ids** parameter.
-The acceptable values for this parameter are:
-
-- 1: Clean 
-- 2: Quarantine 
-- 3: Remove 
-- 6: Allow 
-- 8: UserDefined 
-- 9: NoAction 
+
+Specifies an array of the actions to take for the IDs specified by using the
+**ThreatIDDefaultAction_Ids** parameter. The acceptable values for this parameter are:
+
+- 1: Clean
+- 2: Quarantine
+- 3: Remove
+- 6: Allow
+- 8: UserDefined
+- 9: NoAction
 - 10: Block
```

Comment:
We want to wrap the integers in backticks here, because they're specific values to pass at the command line:

```suggestion
- `1`: Clean
- `2`: Quarantine
- `3`: Remove
- `6`: Allow
- `8`: UserDefined
- `9`: NoAction
- `10`: Block
```